kafka-ui-react-app/src/components/KsqlDb/Query/__test__/Query.spec.tsx

```diff
@@ -65,17 +59,9 @@ describe('Query', () => {
     });
 
     await act(() => {
-      userEvent.paste(
-        within(screen.getByLabelText('KSQL')).getByRole('textbox'),
-        'show tables;'
-      );
-      userEvent.paste(
-        within(
-          screen.getByLabelText('Stream properties (JSON format)')
-        ).getByRole('textbox'),
-        '{"some":"json"}'
-      );
-      userEvent.click(screen.getByRole('button', { name: 'Execute' }));
+      fireEvent.paste(screen.getByLabelText('KSQL'), 'show tables;');
+      fireEvent.paste(screen.getByLabelText('Stream properties:'), 'test');
+      fireEvent.click(screen.getByRole('button', { name: 'Execute' }));
```

Comment:
does userEvent not work here?

Reply:
seems there is a problem related to fieldset tag, via userevent.paste we are getting error in tests but fireevent.paste is working without issues, if there is no mandatory we can let it be fireevent